udata/core/dataset/apiv2.py

```diff
@@ -124,6 +129,18 @@
 })
 
 
+@ns.route('/search', endpoint='dataset_search')
+class DatasetSearch(API):
```

Comment:
Since this class change, we have an issue on v2 orga and reuse search (that we should have seen with tests no?) because we define the OrganizationSearch search twice in apiv2! Could we rollback the class name changes?